src/content-helper/dashboard-page/pages/traffic-boost/store.ts

```
@@ -0,0 +1,735 @@
+				return {
+					...state,
+					suggestionsTab: {
+						...state.suggestionsTab,
+						suggestions: [ action.suggestion, ...state.suggestionsTab.suggestions ],
+					},
+					selectedLink: action.select ? action.suggestion : state.selectedLink,
+				};
+			case 'REMOVE_SUGGESTION':
+				const remainingSuggestions = state.suggestionsTab.suggestions.filter(
+					( suggestion ) => suggestion.uid !== action.suggestion.uid
+				);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)